pkgs/development/python-modules/uplc/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonRelaxDepsHook
+# Python deps
+, frozenlist2
+, python-secp256k1-cardano
+, setuptools
+, poetry-core
+, frozendict
+, cbor2
+, rply
+, pycardano
+}:
+
+buildPythonPackage rec {
```

Comment:
Given the `mainProgram`, should this be `buildPythonApplication`?

Reply:
I don't think so, it's also a library that is imported by other python modules in this PR